

<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Style-Type" content="text/css" />
  <title>Miriam Huntley</title>
  <style type="text/css">code{white-space: pre;}</style>
  <link rel="stylesheet" href="/style.css" type="text/css" />
  <meta name="description" content="Academic page of Miriam Huntley">
  <meta name="keywords" content="Miriam Huntley genomics 3D applied math">
  <script>
   (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
   (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
   m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
   })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
   ga('create', 'UA-91067208-1', 'auto');
   ga('send', 'pageview');
 </script>
</head>
<body>
<div id="miraim-huntley" class="section level1">
<h1>Miriam Huntley</h1>
<p><a href="/index.html">About</a> | <a href="/projects.html">Projects</a> | <a href="/cv.pdf">CV (pdf)</a></p>
<!-- <hr /> -->
    <p>A list of my publications can be found on my <a href="https://scholar.google.com/citations?hl=en&user=Y3A5&user=Y3A5-woAAAAJ">Google Scholar page</a>.</p>  
<div id="projects" class="section level2">
<h2></h2>
<div id="section" class="section level3 project">
<h3></h3>

<p><iframe  src="https://www.youtube.com/embed/dES-ozV65u4" frameborder="0" allowfullscreen></iframe></p>
<div id="3dgenome" class="section level4">
<h4>The 3D Genome at High Resolution</h4>
<p>The three-dimensional structure of the human genome is of great biological importance, but it still is not fully solved. I worked closely with other members of the Aiden group to study the 3D genome, using Hi-C, a next-generation sequencing experiment. We generated and analyzed the largest and highest-resolution Hi-C dataset (to date). The observation of novel high-resolution features in the data led us to develop new scalable analysis techniques. You can find our work in <a href="http://www.cell.com/cell/abstract/S0092-8674(14)01497-4?_returnURL=http%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867414014974%3Fshowall%3Dtrue&cc=y=">our Cell 2014 paper</a> and in followup work.  With <a href="http://suhasrao.com/">Suhas Rao</a>, <a href="http://erez.com/">Erez Lieberman Aiden</a>, and the <a href="http://www.aidenlab.org">Aiden Lab</a>.
</p>
</div>
</div>


<div id="section-1" class="section level3 project">
<h3></h3>
<p><img src="img/chrX.png" alt="Inactive X Chromosome" /></p>
<div id="chrX" class="section level4">
<h4>The 3D Structure of the Inactive X Chromosome</h4>
<p>In human females, one of the two X chromosomes is inactive and adopts an unusual 3D conformation. In this work, we used Hi-C to map this 3D structure in humans, mice, and rhesus macaque. We discovered that a particular element on the inactive X chromosome, DXZ4, is crucial for certain higher-order genomic architecture features. You can find our work in <a href="http://www.pnas.org/content/113/31/E4504.abstract">our PNAS 2016 paper</a>.  With <a href="https://www.researchgate.net/profile/Emily_Darrow">Emily Darrow</a>, <a href="www.bio.fsu.edu/faculty-chadwick.php">Brian Chadwick</a>, <a href="http://erez.com/">Erez Lieberman Aiden</a>, the Chadwick Lab, and the <a href="http://www.aidenlab.org">Aiden Lab</a>.
</p>
</div>
</div>
<div id="section-1" class="section level3 project">
<h3></h3>
<p><img src="img/rmt.png" alt="Random Matrix Theory" /></p>
<div id="rmt" class="section level4">
<h4>Random Matrix Theory and PCA</h4>
<p>I am interested in using tools from random matrix theory to understand the limits of data analysis methods. In particular, random matrix theory (RMT) can be used to study the effectiveness of Principal Component Analysis (PCA), a widely used procedure for uncovering relationships between variables in large datasets. My work has focused on extending RMT tools to describe PCA when applied in slightly altered settings. For example, in many biological analyses, it has become commonplace for `pragmatic' adjustments to PCA to be carried out, wherein the data matrix is transformed by a linear transformation that is chosen to incorporate prior knowledge about the structure of the data. The validity of these adjustments is largely determined ex post facto, with the quality of the results used to justify the validity of the procedure. We developed an RMT based analysis to study these types of transformations, and showed when linear transformations of the data that incorporate prior information improve the signal detectability. Work currently under review. With <a href="https://www.seas.harvard.edu/brenner/Home.html"> Michael Brenner</a> and <a href="http://www.ch.cam.ac.uk/person/ljc37">Lucy Colwell</a>.</p>
</div>
</div>
<div id="section-2" class="section level3 project">
<h3></h3>
<p><img src="img/capacity.png" alt="Capacity" /></p>
<div id="capacity" class="section level4">
<h4>Self-Assembly</h4>
<p>Biology is regularly able to build structures from the bottom-up, and while synthetic experiments have made rapid progress in this space, they are still far behind evolution's innovations. I worked on building theory tools for understanding the limits of self-assembly. Self-assembly (in both biological and synthetic contexts) is typically achieved by engineering binding specificities between the different types of particles, or `species': each species must bind specifically to its target partner, and avoid binding with other off-target binding partners. Complex structures can be built up using these engineered specificities between species, but off-target interactions severely limit the number of species that can be used effectively. In this project I studied what sets the limit for the number of interacting species a system can handle before the crosstalk overwhelms the interactions. Inspired by Shannon's Noisy Channel Coding Theorem, we reframed these systems in terms of information theory and treat desirable interactions between complementary on-target binding partners as the `messages', and off-target binding events as `errors', and derived a `capacity' for these systems: the maximal amount of mutual information that is encodable using engineered binding specificities. You can read more in our 
 <a href="http://www.pnas.org/content/113/21/5841.abstract">our PNAS 2016 paper</a>.
    With  <a href="http://muruganlab.uchicago.edu/">Arvind Murugan</a> and <a href="https://www.seas.harvard.edu/brenner/Home.html"> Michael Brenner</a>.</p></p>
</div>
</div>


</body>
</html>
